src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "user123";
+    private static final String INVALID_LOGIN = null;
+    private static final String VALID_PASSWORD = "password123";
+    private static final String INVALID_PASSWORD = "nvdf";
+    private static final int VALID_AGE = 20;
+    private static final int INVALID_AGE = 17;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private final User user = new User();
+
+    @Test
+    public void registerValidUser_Ok() {
+        user.setLogin(VALID_LOGIN);
+        user.setPassword(VALID_PASSWORD);
+        user.setAge(VALID_AGE);
```

Comment:
let's not duplicate the `user` initialization, and bring it out to the `beforeEach`, where we should also reinitialize it with `new User()`, and let's rename this var to a `validUser` and it the test itself we can then leave just  single invalid field assignment